README.md

```diff
@@ -240,6 +240,37 @@ or
 
 `shasum -a 256 --ignore-missing -c SHA256SUMS` on macOS
 
+### Perplexity (Measuring model quality)
+
+You can pass `--perplexity` as a command line option to measure perplexity over the given prompt.  For more background,
+see https://huggingface.co/docs/transformers/perplexity.  However, in general, lower perplexity is better for LLMs.
+
+#### Measurements
+
+https://github.com/ggerganov/llama.cpp/pull/270 is the unofficial tracking page for now.  llama.cpp is measuring very well
+compared to the baseline implementations.  Quantization has a small negative impact to quality, but, as you can see, running
+13B at q4_0 beats the 7B f16 model by a significant amount.
+```
+Perplexity - model options
```

Comment:
add for which corpus those results are.

Reply:
Probably should also include the context size used, if it can make a meaningful difference.

Reply:
while we are at it, make it a table with models vertical - parameters horizontal

Reply:
Added dataset details/context length.  For the table, I'm not sure we have enough details across the different models for that to work well right now (will be mostly empty for 13B).